### PR TITLE
Update Mixtral.md Blog

### PR DESCRIPTION
**Pull request by saahil1801**
The code for GPTQ in Load Mixtral with GPTQ content below the "Once installed, simply load the GPTQ model with the from_pretrained method:" was incorrect and had not GPTQConfig package imported and used. @anton-l @davanstrien

### CODE REVIEW

**Comment by saahil1801** [comment]
This is the blog needing changes....https://huggingface.co/blog/mixtral

File: mixtral.md [hunk 1]
```diff
@@ -227,7 +227,7 @@ With the 🤗 transformers library, we support out-of-the-box inference with sta
 
 ### Load Mixtral with 4-bit quantization
 
-As demonstrated in the inference section, you can load Mixtral with 4-bit quantization by installing the `bitsandbytes` library (`pip install -U bitsandbytes`) and passing the flag `load_in_4bit=True` to the `from_pretrained` method. For better performance, we advise users to load the model with `bnb_4bit_compute_dtype=torch.float16`. Note you need a GPU device with at least 30GB VRAM to properly run the snippet below.
+As demonstrated in the inference section, you can load Mixtral with 4-bit quantization by installing the `bitsandbytes` library (`pip install -U bitsandbytes`) and passing the flag `load_in_4bit=True` to the `from_pretrained` method. For better performance, we advise users to load the model with `bnb_4bit_compute_dtype=torch.float16` .Note you need a GPU device with at least 30GB VRAM to properly run the snippet below.
```
Comment (younesbelkada) [inline mixtral.md:230]:
```suggestion
As demonstrated in the inference section, you can load Mixtral with 4-bit quantization by installing the `bitsandbytes` library (`pip install -U bitsandbytes`) and passing the flag `load_in_4bit=True` to the `from_pretrained` method. For better performance, we advise users to load the model with `bnb_4bit_compute_dtype=torch.float16`. Note you need a GPU device with at least 30GB VRAM to properly run the snippet below.
```

**Review by younesbelkada — COMMENTED** [review]
Thanks ! Hmm are you sure about that? One should be able to load any gptq model without passing the GPTQ Config, this seems to educate users on how to use GPTQ+exllama, maybe this can go on a separate section? What do you think?

**Comment by saahil1801** [comment]
Yes , it can go in a seperate section definitely @younesbelkada

**Pull request closed without merging**